src/LanguageServer.php

```diff
@@ -190,6 +191,7 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->projectIndex = new ProjectIndex($sourceIndex, $dependenciesIndex, $this->composerJson);
             $stubsIndex = StubsIndex::read();
             $this->globalIndex = new GlobalIndex($stubsIndex, $this->projectIndex);
+            $initializationOptions = $initializationOptions ?? new Options;
```

Comment:
Oh sorry, I think I missed the type annotation because I looked at the closure `use`. But shouldn't this be the same object as passed to `didChangeConfiguration`? And will this break passing custom properties from the client in `initializationOptions`?